[2024-01-05T22:12:19.404+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.test_model_naive scheduled__2024-01-03T14:30:00+00:00 [queued]>
[2024-01-05T22:12:19.411+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.test_model_naive scheduled__2024-01-03T14:30:00+00:00 [queued]>
[2024-01-05T22:12:19.411+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-05T22:12:19.411+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-05T22:12:19.412+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-05T22:12:19.421+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): test_model_naive> on 2024-01-03 14:30:00+00:00
[2024-01-05T22:12:19.424+0300] {standard_task_runner.py:55} INFO - Started process 206491 to run task
[2024-01-05T22:12:19.431+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'test_model_naive', 'scheduled__2024-01-03T14:30:00+00:00', '--job-id', '568', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmpt9fk2pfr']
[2024-01-05T22:12:19.432+0300] {standard_task_runner.py:83} INFO - Job 568: Subtask test_model_naive
[2024-01-05T22:12:19.534+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.test_model_naive scheduled__2024-01-03T14:30:00+00:00 [running]> on host prod-srv
[2024-01-05T22:12:19.616+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=test_model_naive
AIRFLOW_CTX_EXECUTION_DATE=2024-01-03T14:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-03T14:30:00+00:00
[2024-01-05T22:12:19.617+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-05T22:12:19.617+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/igor/mlops_4/scripts/test_model.py naive']
[2024-01-05T22:12:19.622+0300] {subprocess.py:86} INFO - Output:
[2024-01-05T22:12:21.557+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-05T22:12:21.598+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=test_model_naive, execution_date=20240103T143000, start_date=20240105T191219, end_date=20240105T191221
[2024-01-05T22:12:21.613+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-05T22:12:21.635+0300] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
